README.md

```diff
@@ -100,6 +107,19 @@ show_hints
 
 The <code>show_hints</code> command will display all the hints you have requested to that point, in the task you have started.
 
+_Ending a task_
+<pre>
+end_task
+</pre>
+
+or 
+
+<pre>
+start_task 2
+</pre>
+
+The <code>end_task</code> command will end the task you are currently on and score you. This will also happen if you move onto the a new task with the <code>start_task</code> command. When you end a task in one of these ways, you will be scored 100 points deducted the amount of hints you have displayed for that task at a value of -10 each. This is customizable through the task yamls.

```

Comment:
Would 'minus' fit better than 'deducted' here? I'm not sure I'd mention the `10` value here since it's just a placeholder. Maybe say it's usually 10 but it is customizable.